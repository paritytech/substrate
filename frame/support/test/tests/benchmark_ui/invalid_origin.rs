use frame_benchmarking::v2::*;
#[allow(unused_imports)]
use frame_support_test::Config;
use frame_support_test::Call;

#[benchmarks]
mod benches {
	use super::*;

	#[benchmark]
	fn bench() {
		#[extrinsic_call]
		thing(1);
	}
}

fn main() {}
