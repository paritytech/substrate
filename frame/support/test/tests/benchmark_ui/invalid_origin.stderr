error[E0599]: no variant or associated item named `new_call_variant_thing` found for enum `Call` in the current scope
 --> tests/benchmark_ui/invalid_origin.rs:6:1
  |
6 | #[benchmarks]
  | ^^^^^^^^^^^^^ variant or associated item not found in `Call<T>`
  |
  = note: this error originates in the attribute macro `benchmarks` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0277]: the trait bound `<T as frame_support_test::Config>::RuntimeOrigin: From<{integer}>` is not satisfied
 --> tests/benchmark_ui/invalid_origin.rs:6:1
  |
6 | #[benchmarks]
  | ^^^^^^^^^^^^^ the trait `From<{integer}>` is not implemented for `<T as frame_support_test::Config>::RuntimeOrigin`
  |
  = note: required for `{integer}` to implement `Into<<T as frame_support_test::Config>::RuntimeOrigin>`
  = note: this error originates in the attribute macro `benchmarks` (in Nightly builds, run with -Z macro-backtrace for more info)
